lib/src/node-sass/render.test.ts

```diff
@@ -141,6 +141,19 @@ describe('render', () => {
         });
       });
     });
+
+    it('renders indented syntax if indentedSyntax is enabled', done => {
+      render(
+        {
+          data: 'a\n\tb: c',
+          indentedSyntax: true,
+        },
+        (_, result) => {
+          expectEqualIgnoringWhitespace(result!.css.toString(), 'a {b: c;}');
+          done();
+        }
+      );
+    });
   });
```

Comment:
There should be a few more tests here to cover the syntaxes:
- `.sass` file
- `.scss` file
- string with indented syntax (done)
- string with scss syntax

Reply:
Thank you for your comment! I just wanted to make sure I'm not misunderstanding you. Wouldn't the test `renders a file from a relative path` cover the `.scss` case, and wouldn't the test `renders a string` cover the "string with scss syntax". Or do you prefer creating additional, explicit, test cases?

Reply:
Good point, I'm fine with not adding explicit cases.